examples/kitchensink-ts/src/admin/app.example.tsx

```diff
@@ -0,0 +1,52 @@
+import theme from './extensions/theme';
+
+type Config = {
```

Comment:
`interface` should be used on this file (interfaces represents object "shapes", can be augmented, extended, etc...) whereas `type` should be used for aliases and intersections

(in most case `interface` should be used and `exported`)

Reply:
👍  this file is going to be removed anyway I was using it in order to test if everything was working correctly.

Thanks for the comment !